pkg/sanity/controller.go

```diff
@@ -569,9 +570,20 @@ var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *TestCo
 				Skip("Snapshot not supported")
 			}
 
-			By("creating a snapshot")
+			By("creating a volume")
 			vol1Req := MakeCreateVolumeReq(sc, UniqueString("sanity-controller-source-vol"))
-			snap, _ := r.MustCreateSnapshotFromVolumeRequest(context.Background(), vol1Req, UniqueString("sanity-controller-snap-from-vol"))
+			vol1 := r.MustCreateVolume(context.Background(), vol1Req)
```

Comment:
MustCreateSnapshotFromVolumeRequest is called in multiple places. Can you modify that instead of replacing it at this one place?

Reply:
Only this snapshot is actually used for creating volume. So I think only this one needs fix. All other places are fine.

Reply:
@xing-yang Could you PTAL again? thanks.